src/layouts/partials/footer.html

```diff
@@ -2,43 +2,27 @@
-                <a href="{{ "/press/" }}" class="footerblocklink transition">{{ T "nav_in_the_press" }}</a>
-                <a href="{{ "/recruiting/"  }}" class="footerblocklink transition">{{ T "nav_become_contributor" }}</a>
-            </div>
-            <div class="footerrowscolumn1 w-col w-col-2 w-col-medium-2">
-                <a href="https://docs.decred.org/research/overview/" target="_blank" rel="noopener noreferrer" class="footerblocklink transition" >{{ T "nav_tech_overview" }}</a>
-                <a href="{{ "/brief/"  }}" rel="noopener noreferrer" class="footerblocklink transition">{{ T "nav_business_brief" }}</a>
-                <a href="{{ "roadmap/" | relLangURL }}" target="_blank" rel="noopener noreferrer" class="footerblocklink transition">{{ T "footer_roadmap" }}</a>
-            </div>
-            <div class="footerblockstats footerrowscolumn1 w-clearfix w-col w-col-2 w-col-medium-2">
-                <div class="footerblockstatsrow w-clearfix">
-                    {{ with $current_release }}
-                        <a id="footerRelease" class="backgroundturquoise borderradius4 dropshadow footerblockstatslink" href="{{ .release_url_base }}/{{ .current_version }}" target="_blank" rel="noopener noreferrer">{{ .current_version }}</a>
-                    {{ end }}
-                    <div class="footerblockstatsname">{{ T "general_release_stable" }}</div>
-                </div>
-                <div class="footerblockstatsrow w-clearfix"><a id="footerDownloads" class="backgroundwashedblue borderradius4 dropshadow footerblockstatslink" href="{{ "downloads/" | relLangURL }}">52k {{ T "general_total" }}</a>
-                    <div class="footerblockstatsname">{{ T "general_downloads" }}</div>
-                    <div id="explorer-links">
-                      <a href="https://dcrdata.decred.org/" target="_blank" rel="noopener noreferrer" class="footer-inline-el footerblocklink transition">{{ T "nav_dcrdata" }}</a>
-                    </div>
-                </div>
-            </div>
-            <div class="footerblockicon footerrowscolumn1 w-clearfix w-col w-col-2 w-col-medium-2">
-                {{ range $.Site.Params.social_media_links }}
-                    <a target="_blank" rel="noopener noreferrer" href="{{ .url }}" data-label="{{ .label }}" class="icon w-inline-block {{ .id }}"></a>
-                {{ end }}
-                <div class="backgroundlightblue borderradius4 colorblue dropshadow font12 footerblockindicator opacity000 transition"></div>
-            </div>
+    <div class="footer__container width1100">
+        <ul class="footer__menu">
+            <li><a href="{{ "brand/" | relLangURL }}">{{ T "nav_brand_resources" }}</a></li>
+            <li><a href="{{ "mailto:press@decred.org" }}">Press Contacts</a></li>
+            <li><a href="{{ "https://docs.decred.org/about/license/" | relLangURL }}">Licensing Information</a></li>
```

Comment:
Two points here:

1. External URLs do not need to be run through `relLangURL`
1. This link actually goes to the licensing information specifically for dcrdocs. This is not a project-wide license.

Reply:
rel="noopener noreferrer" on all links.